.eslintrc.js

```diff
@@ -1,15 +1,24 @@
 module.exports = {
     "extends": "airbnb-base",
     "plugins": [
-        "import"
+        "import",
+        "chai-friendly",
     ],
     "globals": {
       "window": true,
       "document": true,
       "jQuery": true,
+      "Zoolander": true,
     },
     "rules": {
       // Errors.
       "no-param-reassign": ["error", { "props": true, "ignorePropertyModificationsFor": ["$"] }],
+      "no-unused-expressions": 0,
```

Comment:
Nevermind, I see it was part of the `chai-friendly` config:

https://www.npmjs.com/package/eslint-plugin-chai-friendly